### PR TITLE
Style cleanup

### PR DESCRIPTION
**Pull request by rogermc2**
Problem styles fixed and updated to use new freetype.

### CODE REVIEW

File: examples/freetype_gl/single_character/src/main_loop.adb [hunk 1]
```diff
@@ -43,21 +39,19 @@ procedure Main_Loop (Main_Window : in out Glfw.Windows.Window) is
    Colour_Location       : GL.Uniforms.Uniform;
    Projection_Location   : GL.Uniforms.Uniform;
    Texture_Location      : GL.Uniforms.Uniform;
-   Position_Location     : GL.Attributes.Attribute;
    Projection_Matrix     : Singles.Matrix4;
 
-   Back_Colour   : constant GL.Types.Colors.Color := (0.3, 0.6, 0.6, 1.0);
+   Back_Colour    : constant GL.Types.Colors.Color := (0.3, 0.6, 0.6, 1.0);
+   Main_Exception : Exception;
```
Comment (flyx) [inline examples/freetype_gl/single_character/src/main_loop.adb]: Why is this necessary? Declaring an exception in a package body is very bad style, as no-one can catch it from outside. There should be a better way to fix compiler warnings than introducing this exception.

**Comment by rogermc2** [comment]
OK. Fixed.

**Comment by flyx** [comment]
`Main_Exception` is still there.

**Comment by rogermc2** [comment]
It looks like I fixed it with commit 28b9f6c but then seem somehow  seem to have lost it.
Main_Exception  is not in my local copy.
git status shows the following:
~~~
On branch style_cleanup
Your branch is ahead of 'upstream/master' by 178 commits.
  (use "git push" to publish your local commits)
Untracked files:
  (use "git add <file>..." to include in what will be committed)

	examples/ogl_tutorials/04_coloured_cube/tutorial_04

nothing added to commit but untracked files present (use "git add" to track)
:/Ada_Source/OpenGLAda Roger$git push origin style_cleanup
Everything up-to-date
~~~
Can you advise me how to fix it?

**Comment by rogermc2** [comment]
I don't understand,  git status shows.
~~~
git status
On branch style_cleanup
Your branch is ahead of 'upstream/master' by 178 commits.
  (use "git push" to publish your local commits)
Untracked files:
  (use "git add <file>..." to include in what will be committed)

	examples/ogl_tutorials/04_coloured_cube/tutorial_04

nothing added to commit but untracked files present (use "git add" to track)
:/Ada_Source/OpenGLAda Roger$git push origin style_cleanup
Everything up-to-date
~~~

**Comment by rogermc2** [comment]
OK, found it. I removed it from `Texture_Manager.adb` but didn't realize that it was also in `Main_Loop`
Will fix!
